src/Interface/SteeringDisplay.cpp

```diff
@@ -67,7 +67,7 @@
 #define THROTTLE_RAW_Y 210
 
 // turn signals
-#define TURN_FLASHING_INTERVAL	500
+#define TURN_FLASHING_INTERVAL	490
```

Comment:
This is good, and I'm glad it works. However, can you think of a way to synchronize them by having some sort of variable that's shared between graphics and main? For example, a boolean for blink on/blink off?